docs/tutorials/multi-product-formula.ipynb

```diff
@@ -0,0 +1,2626 @@
+    "\n",
+    "\n",
+    "plt.title(\n",
+    "    f\"Expectation values for (ZZ,{(L//2-1, L//2)}) at time {total_time} for the different methods \"\n",
+    ")\n",
+    "plt.xlabel(\"Method\")\n",
+    "plt.ylabel(\"Expectation Value\")\n",
+    "plt.legend(loc=\"upper center\", bbox_to_anchor=(0.5, -0.2), ncol=2)\n",
+    "plt.grid(alpha=0.1)\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04200168",
+   "metadata": {},
+   "source": [
+    "When executing circuits on hardware we may encounter additional challenges in obtaining accurate expectation values due to the presence of hardware noise. This is not accounted for in the MPF formalism and could play against the MPF solution. For example, this could be the reason for the failure of the dynamic coefficients to provide a better estimate of the expectation value compared to the approximate static coefficient in the plot. That is, the approximate evolver, which simulates the approximate circuit, does not accurately reflect the results obtained by executing the approximate circuits in the presence of hardware noise. For these reasons, it is recommended to combine different error mitigation techniques in order to get results as close as possible to the ideal values for each of the product formulas. This will show consistent benefits from the MPF approach.\n",
+    "Overall, the approximate static coefficients still give a more accurate solution than the product formula with higher number of trotter steps with the same amount of Trotter error in the noiseless setting.\n",
+    "\n",
+    "It is also important to note that in this example which reproduces the experiment in Ref.  [\\[3\\]](#Reference3), the time point $t=3$ is beyond the limit at which it is expected that the PF with $k=2$ behaves well, that is $t/k>1$ as discussed in this [guide](https://qiskit.github.io/qiskit-addon-mpf/how_tos/choose_trotter_steps.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78e91ddc",
+   "metadata": {},
+   "source": [
+    "## References\n",
+    "\n",
+    "[\\[1\\]](#Reference1) [Vazquez, A. C., Egger, D. J., Ochsner, D., & Woerner, S. (2023). Well-conditioned multi-product formulas for hardware-friendly Hamiltonian simulation. Quantum, 7, 1067.](https://quantum-journal.org/papers/q-2023-07-25-1067/)\n",
+    "\n",
+    "[\\[2\\]](#Reference2) [Zhuk, S., Robertson, N. F., & Bravyi, S. (2024). Trotter error bounds and dynamic multi-product formulas for Hamiltonian simulation. Physical Review Research, 6(3), 033309.](https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.6.033309)\n",
+    "\n",
+    "[\\[3\\]](#Reference3) [Robertson, N. F., et al. (2024). Tensor network enhanced dynamic multiproduct formulas. arXiv preprint arXiv:2407.17405.](https://arxiv.org/abs/2407.17405)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9364288",
+   "metadata": {},
+   "source": [
+    "© IBM Corp. 2025"
+   ]
+  }
```

Comment:
We don't add the copyright statement explicitly. The app already adds the statement when it's relevant, like downloading the notebook and printing it.

```suggestion
  }
```